extensions/wikia/AdminDashboard/AdminDashboardController.class.php

```diff
@@ -33,6 +33,8 @@ public function chrome() {
 		$this->adminDashboardUrl = $this->wg->Title->getFullURL( [ 'tab' => $this->tab ] );
 		$this->adminDashboardUrlGeneral = $this->wg->Title->getFullURL( [ 'tab' => 'general' ] );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)